packages/webapp/pages/posts/[id]/index.tsx

```diff
@@ -203,10 +188,26 @@ export async function getStaticProps({
       POST_BY_ID_STATIC_FIELDS_QUERY,
       { id },
     );
+
+    const post = initialData.post as Post;
+    const seo: NextSeoProps = {
+      canonical: post?.slug ? `${webappUrl}posts/${post.slug}` : undefined,
+      title: getTemplatedTitle(seoTitle(post)),
+      description: getSeoDescription(post),
+      openGraph: {
+        images: [{ url: `https://og.daily.dev/api/posts/${post?.id}` }],
+        article: {
+          publishedTime: post?.createdAt,
+          tags: post?.tags,
+        },
+      },
+    };
+
```

Comment:
This is basically how our server-side rendering would look like once we migrated to `app` router, and we will render the SEO metadata from there. The only difference is the generated SEO is utilized somewhere else while being passed as a prop in here.